conftest.py

```diff
@@ -1,36 +1,31 @@
 import os
-import time
-from asyncio import timeout
-from fileinput import close
-
 import pytest
-from playwright.sync_api import Playwright, expect
 from pytest_playwright.pytest_playwright import browser
+from utils import webshop_config
+from utils.webshop_config import PASSWORD
+from utils.webshop_login_helpers import webshop_login
 
-import utils.secret_config
-from utils.secret_config import PASSWORD
-
+no_browser= True
 
 # this is causing that if password from githhub is not available, local password will be used, this allows to switch
 # between local and remote run
 try:
     PASSWORD = os.environ['PASSWORD']
 except KeyError:
```

Comment:
Instead of `try ... except KeyError` you could use [get method](https://docs.python.org/3.10/library/stdtypes.html#dict.get) of dictionary with default value.